src/scval_conversions.rs

```diff
@@ -1,15 +1,10 @@
-use crate::{
-    AccountId, ScBigInt, ScMap, ScMapEntry, ScObject, ScStatic, ScStatus, ScSymbol, ScVal, ScVec,
-};
+use crate::{AccountId, ScMap, ScMapEntry, ScObject, ScStatic, ScStatus, ScSymbol, ScVal, ScVec};
 
 #[cfg(all(not(feature = "std"), feature = "alloc"))]
 extern crate alloc;
 #[cfg(all(not(feature = "std"), feature = "alloc"))]
 use alloc::{string::String, vec, vec::Vec};
```

Comment:
Can you add conversions for u128 and i128 here similar to the 64bit ones?

Reply:
Why: ScVal to and from conversions are needed in the SDK for all types as there are things like message signing in tests that rely on them. Without this the types won't be usable in UDTs that require all their fields to be convertible.

Reply:
See line 113+.

Reply:
added

Reply:
Thanks!

Reply:
Looks like clippy has issues with the casts in the last commit.